desktop/renderer-app/src/tasks/init-register-apps.ts

```diff
@@ -1,6 +1,7 @@
 import { WindowManager } from "@netless/window-manager";
 import AppSlide, { addHooks as addHooksSlide } from "@netless/app-slide";
 import path from "path";
+import "@netless/fastboard-react";
```

Comment:
I've had @hyrious make a new version of `fastboard-react` which supports `import { register } from "@netless/fastboard-react";`. This should make the code a bit more explicit.